app/views/auth/sessions/new.html.haml

```diff
@@ -1,30 +1,22 @@
 - content_for :page_title do
   = t('auth.login')
 
-- content_for :header_tags do
-  = render partial: 'shared/og'
+.simple_form.alternative-login
+  %h4= 'Redirecting...'
 
-- unless omniauth_only?
-  = simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
-    %h1.title= t('auth.sign_in.title', domain: site_hostname)
-    %p.lead= t('auth.sign_in.preamble_html', domain: site_hostname)
-    .fields-group
-      - if use_seamless_external_login?
-        = f.input :email, autofocus: true, wrapper: :with_label, label: t('simple_form.labels.defaults.username_or_email'), input_html: { 'aria-label': t('simple_form.labels.defaults.username_or_email') }, hint: false
-      - else
-        = f.input :email, autofocus: true, wrapper: :with_label, label: t('simple_form.labels.defaults.email'), input_html: { 'aria-label': t('simple_form.labels.defaults.email') }, hint: false
-    .fields-group
-      = f.input :password, wrapper: :with_label, label: t('simple_form.labels.defaults.password'), input_html: { 'aria-label': t('simple_form.labels.defaults.password'), autocomplete: 'current-password' }, hint: false
-
-    .actions
-      = f.button :button, t('auth.login'), type: :submit
-
-- if devise_mapping.omniauthable? && resource_class.omniauth_providers.any?
```

Comment:
Do we need to delete all of this?  Couldn't you just add your script under the `.simple_form.alternative-login` class?  I wonder if this might make managing upstream conflicts easier?